2020-InfiniteRecharge/src/main/java/frc/robot/subsystems/PowerCell.java

```diff
@@ -169,12 +217,14 @@ public void storage(){
 	public void intake() {
         if(oi.copilot.getRawButton(1))
         {
+            motortemp = intakeMotor.getMotorTemperature();
             intakevelocity = intakeEncoder.getVelocity();
             intakeMotorOutputCurrent = intakeMotor.getOutputCurrent();
             intakeMotorPID.setReference(intakeRpms, ControlType.kVelocity);
         }
         else
         {
+            motortemp = intakeMotor.getMotorTemperature();
             intakevelocity = intakeEncoder.getVelocity();
             intakeMotorOutputCurrent = intakeMotor.getOutputCurrent();
```

Comment:
These 3 lines are duplicated - both copies could move outside of the `if` statement.